library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
> I would suggest an additional role that is included via dependency in meta/main.yml from common, masters, and nodes roles.

I'm trying that. Apparently there is a side-effect that the dependency gets loaded/executed 3 times... is that expected behavior? It makes the playbook run slower.